### ISSUE TITLE
[bitnami/mariadb-galera]MariaDB Galera 10.11.2 install error

### ISSUE THREAD

**Issue by FawenYo** [bitnami/charts#16090]
### Name and Version

bitnami/mariadb-galera:10.11.2-debian-11-r14

### What architecture are you using?

amd64

### What steps will reproduce the bug?

(Both amd64 and arm64 have the same issue)
1. Install with helm command `helm install mariadb-galera bitnami/mariadb-galera --set image.tag=10.11.2-debian-11-r14`

### Are you using any custom parameters or values?

`--set image.tag=10.11.2-debian-11-r14`

### What is the expected behavior?

Database startup normally

### What do you see instead?

```
mariadb 12:18:47.60 Welcome to the Bitnami mariadb-galera container
mariadb 12:18:47.60 Subscribe to project updates by watching https://github.com/bitnami/containers
mariadb 12:18:47.60 Submit issues and feature requests at https://github.com/bitnami/containers/issues
mariadb 12:18:47.60 
mariadb 12:18:47.60 INFO  ==> ** Starting MariaDB setup **
mariadb 12:18:47.62 INFO  ==> Validating settings in MYSQL_*/MARIADB_* env vars
mariadb 12:19:03.81 INFO  ==> Initializing mariadb database
mariadb 12:19:03.85 INFO  ==> Found mounted configuration directory
mariadb 12:19:03.86 INFO  ==> Updating 'my.cnf' with custom configuration
mariadb 12:19:03.87 INFO  ==> Setting slow_query_log option
mariadb 12:19:03.87 INFO  ==> Setting long_query_time option
mariadb 12:19:03.88 INFO  ==> Setting wsrep_node_name option
mariadb 12:19:03.89 INFO  ==> Setting wsrep_node_address option
mariadb 12:19:11.97 INFO  ==> Setting wsrep_sst_auth option
mariadb 12:19:50.60 
mariadb 12:19:50.60 Welcome to the Bitnami mariadb-galera container
mariadb 12:19:50.60 Subscribe to project updates by watching https://github.com/bitnami/containers
mariadb 12:19:50.60 Submit issues and feature requests at https://github.com/bitnami/containers/issues
(Repeat the same log)
```

The pod just keep restarting

**Comment by carrodher** [comment]
Are you able to reproduce the issue without using a specific image version? I mean, using the default one. In the same way, can you add the `image.debug=true` parameter so the application shows more info in the logs during the initialization process?

**Comment by FawenYo** [comment]
It works successfully if using default image (10.6.12-debian-11-r16), so the issue should comes with newer mariadb-galera image.

And below is the debug message part:
```
[38;5;6mmariadb [38;5;5m08:16:27.98 [0m[38;5;5mDEBUG[0m ==> Setting wsrep_sst_auth to 'mariabackup:SwlXbsNlCC' in mariadb configuration file /opt/bitnami/mariadb/conf/my.cnf
[38;5;6mmariadb [38;5;5m08:16:36.03 [0m[38;5;5mDEBUG[0m ==> Cleaning data directory to ensure successfully initialization
Installing MariaDB/MySQL system tables in '/bitnami/mariadb/data' ...

Installation of system tables failed!  Examine the logs in
/opt/bitnami/mariadb/logs/mysqld.log or /bitnami/mariadb/data for more information.

The problem could be conflicting information in an external
my.cnf files. You can ignore these by doing:

    shell> /opt/bitnami/mariadb/bin/mysql_install_db --defaults-file=~/.my.cnf

You can also try to start the mariadbd daemon with:

    shell> /opt/bitnami/mariadb/sbin/mariadbd --skip-grant-tables --general-log &

and use the command line tool /opt/bitnami/mariadb/bin/mariadb
to connect to the mysql database and look at the grant tables:

    shell> /opt/bitnami/mariadb/bin/mariadb -u root mysql
    MariaDB> show tables;

Try '/opt/bitnami/mariadb/sbin/mariadbd --help' if you have problems with paths.  Using
--general-log gives you a log in /bitnami/mariadb/data that may be helpful.

The latest information about mysql_install_db is available at
https://mariadb.com/kb/en/installing-system-tables-mysql_install_db
You can find the latest source at https://downloads.mariadb.org and
the maria-discuss email list at https://launchpad.net/~maria-discuss

Please check all of the above before submitting a bug report
at https://mariadb.org/jira
```

**Comment by carrodher** [comment]
That can be definitely something related to the application version, please note Helm charts only support one major version or branch. In those assets whose main product support different branches, the replacement of the container image by a different branch can or can't work, but it is not something we test as part of the release process so we can't guarantee that it works nor provide support about this non-tested scenario.

**Comment by carrodher** [comment]
Being said that we currently have an internal task to make MariaDB and MariaDB Galera use 10.11 as the default version since it's the version considered by the upstream as the long-term support (LTS)

**Comment by FawenYo** [comment]
OK. Any ETA for 10.11 support? or maybe there can be some modification I can do to let 10.11 version work?

**Comment by carrodher** [comment]
No ETA sorry, it depends on other initiatives we are working on. We will update this thread once completed so you are notified about the version bump

**Comment by dgomezleon** [comment]
Hi @FawenYo ,

MariaDB-Galera 10.11 is now available #16160 .
